docs/docs-content/clusters/edge/edgeforge-workflow/palette-canvos/build-installer-iso.md

```diff
@@ -136,14 +143,24 @@ If you do not include content bundle in your Edge Installer ISO, you can still b
 a disconnected Edge host instance via [Local UI](../../local-ui/local-ui.md). For more information, refer to
 [Upload Content Bundle](../../local-ui/cluster-management/upload-content-bundle.md).
 
-8. Refer to [Build Content Bundle](build-content-bundle.md) to learn how to build content bundles for your ISO image.
+9. Refer to [Build Content Bundle](build-content-bundle.md) to learn how to build content bundles for your ISO image.
    Since you are including the content bundle in the Installer ISO, you should choose either the zst format or the tar
    format for the content bundle. Do not build the content bundle as an ISO image.
 
-9. When the content bundle build finishes, the output will be in a directory named **content-XXXXXX**, where XXXXXX is a
-   random alphanumerical string. Inside the directory is the content bundle file.
+   :::info
+
+   If you are including a content bundle in your ISO build and you are embedding a public key in your ISO, you do not
```

Comment:
@lennessyy I read this info box a few times and I'm still a bit confused 😅 
If I understand correctly, are you saying that the content bundle does not need to be signed by the included public key's counterpart private key? Is there a way we could simplify the statement if I understood correctly?

Reply:
Yes, the content bundle does not need to be signed if it is part of the ISO build. It only needs to be signed if it is uploaded post-installation. 

 I will think of a more straightforward way to put this.